mleap-runtime/src/main/scala/ml/combust/mleap/bundle/ops/feature/OneHotEncoderOp.scala

```diff
@@ -30,18 +30,19 @@ class OneHotEncoderOp extends MleapOp[OneHotEncoder, OneHotEncoderModel] {
 
     override def load(model: Model)
                      (implicit context: BundleContext[MleapContext]): OneHotEncoderModel = {
+      val dropLast = model.value("drop_last").getBoolean
       if (model.getValue("size").nonEmpty) {
         // Old version of 1HE.
+        val handleInvalid = if (dropLast) HandleInvalid.Keep else HandleInvalid.default
```

Comment:
You hard-code `attributes.append(('drop_last', True))` in `data.py`, so this removes the choice between `HandleInvalid.Keep` and `HandleInvalid.Error`.

Instead, we should just read whatever the bundle has inside of it, and have the Python class serialize:
```
if transformer.handle_unknown == 'ignore':
    attributes.append(('handle_invalid', 'keep'))
else:
    attributes.append(('handle_invalid', 'error'))
attributes.append(('drop_last', True))
```

Reply:
You're right, I've amended the code.